services/core/cards-publication/src/main/modeling/swagger.yaml

```diff
@@ -505,8 +505,8 @@ definitions:
           type: string
         example: ["Dispatcher","Planner"]
         description: >-
-          List of entities that have to respond (should be a subset of entitiesAllowedToRespond). If present and not
-          empty, this list will be used for display in the card detail header instead of entitiesAllowedToRespond.
+          List of entities that have to respond. If present and not empty, this list will be used for display in the
```

Comment:
```suggestion
          List of entities that have to respond. It will be taken into account in addition to entitiesAllowedToRespond to determine if a user can send a response. If present and not empty, this list will be used for display in the
```

Reply:
Done.